swap/src/network/transport.rs

```diff
@@ -23,10 +21,8 @@ pub fn build_clear_net(id_keys: &identity::Keypair) -> Result<SwapTransport> {
 
     let tcp = TokioTcpConfig::new().nodelay(true);
     let dns = TokioDnsConfig::system(tcp)?;
-    let websocket = WsConfig::new(dns.clone());
 
-    let transport = websocket
-        .or_transport(dns)
+    let transport = dns
```

Comment:
Note: you will need to update the config wizard and remove the setting which says we are listening for ` /ip4/0.0.0.0/tcp/9940/ws`